demisto_sdk/commands/common/update_id_set.py

```diff
@@ -94,6 +94,29 @@
 ]
 
 
+def is_dict_have_alternative_key(dict_to_search: dict) -> bool:
+    """
+        Check if a key that ends with "_x2" exists in the dict (including inner levels)
+        Args:
+            dict_to_search (dict): the dict to search in
+
+        Returns: True if found such a key, else False
+
+        """
```

Comment:
align with the opening quotes
```suggestion
    """
```